hedyorg/hedy
# 🪲 Fix highlight for the `add` and `remove` commands

The grammar rule for the `add` and  `remove` commands didn't allow for multiple words to be used. Fix that and add some tests.

Fixes #5074

**How to test**

1. Go to level 3 and paste this code: `remove sour cream from sauces`.
2. Also check this code: `add sour cream to sauces`
